### PR TITLE
added stride testnet

### PR DESCRIPTION
**Pull request by sampocs**
### Description
Added stride testnet

### Backward compatibility
Yes

### Testing
N/A

### CODE REVIEW

File: chains/stridetestnet/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+chainId: stride-internal-1
+deployer:
+  name: Stride Labs
+  url: https://www.stride.zone
+displayName: Stride
+domainId: 1651
+gasCurrencyCoinGeckoId: stride
+grpcUrls:
+  - http: stride-testnet-grpc.polkachu.com:12290
+isTestnet: false
```
Comment (jmrossy) [inline chains/stridetestnet/metadata.yaml:21]: This should be true

File: chains/stridetestnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    url: https://testnet.ping.pub/stride
+blocks:
+  confirmations: 1
+  estimateBlockTime: 5
+  reorgPeriod: 1
+chainId: stride-internal-1
+deployer:
+  name: Stride Labs
+  url: https://www.stride.zone
+displayName: Stride
```
Comment (jmrossy) [inline chains/stridetestnet/metadata.yaml:16]: Should this be Stride Testnet?

File: .changeset/stupid-rice-retire.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': patch
```
Comment (jmrossy) [inline .changeset/stupid-rice-retire.md:2]:
```suggestion
'@hyperlane-xyz/registry': minor
```

**Review by jmrossy — COMMENTED** [review]
Thanks, just a few things to fix

File: chains/stridetestnet/metadata.yaml (hunk 2 above)
Comment (sampocs) [reply chains/stridetestnet/metadata.yaml]: resolved [here](https://github.com/hyperlane-xyz/hyperlane-registry/pull/141/commits/dd7a361add9cee0f80fdcdbf5a28a21a39e54149)

File: .changeset/stupid-rice-retire.md (hunk 3 above)
Comment (sampocs) [reply .changeset/stupid-rice-retire.md]: resolved [here](https://github.com/hyperlane-xyz/hyperlane-registry/pull/141/commits/dd7a361add9cee0f80fdcdbf5a28a21a39e54149)

**Review by jmrossy — APPROVED** [review]

**Comment by avious00** [comment]
thanks @sampocs we'll get this in shortly!

**Pull request merged**